caddy/module.go

```diff
@@ -162,6 +169,21 @@ func (f *FrankenPHPModule) ServeHTTP(w http.ResponseWriter, r *http.Request, _ c
 		}
 	}
 
+	if r.TLS != nil {
+		if env == nil {
+			env = make(frankenphp.PreparedEnv)
+		}
+		env["HTTPS\x00"] = "on"
+
+		if protocol, ok := tlsProtocolStrings[r.TLS.Version]; ok {
+			env["SSL_PROTOCOL\x00"] = protocol
```

Comment:
Maybe log an error if the version isn't found (this should not happen and we should notice and this)